.gitignore

```diff
@@ -1,2 +1,7 @@
-./composer-cli
+composer-cli
```

Comment:
The former only matches the binary in the top of the workdir, the latter matches any file, or directory, named composer-cli so it messes up `./cmd/composer-cli/`

Reply:
I also thought so and I'll check again but in my case `git` showed `./composer-cli` as _changed_ without this

Reply:
ooh - now I got it - I also mixed up syntaxes :-| my IDE now showed me the correct syntax `/composer-cli` ;)